### PR TITLE
privateAccess_ARMTemplate

### PR DESCRIPTION
**Pull request by v-gsapaharam**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

**Comment by v-gsapaharam** [comment]
Hi Mahender, could you please review this PR.

**Comment by shailja-khurana** [comment]
Looks good to me

**Comment by v-gsapaharam** [comment]
> Looks good to me

Thankyou

File: quickstarts/microsoft.databricks/databricks-all-in-one-template-for-vnet-injection-privateaccess/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,292 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "metadata": {
```
Comment (bmoore-msft) [inline quickstarts/microsoft.databricks/databricks-all-in-one-template-for-vnet-injection-privateaccess/azuredeploy.json:4]: remove this or add the bicep file to the sample

**Review by bmoore-msft — CHANGES_REQUESTED** [review]
Check the pipeline results for changes that need to be made...  Also, remove the unused pngs

File: quickstarts/microsoft.databricks/databricks-all-in-one-template-for-vnet-injection-privateaccess/azuredeploy.json (hunk 1 above)
Comment (v-gsapaharam) [reply quickstarts/microsoft.databricks/databricks-all-in-one-template-for-vnet-injection-privateaccess/azuredeploy.json:4]: Sorry for the delay, here. will try to incorporate the review comments at the earliest.

Comment (v-gsapaharam) [reply quickstarts/microsoft.databricks/databricks-all-in-one-template-for-vnet-injection-privateaccess/azuredeploy.json:4]: >Thanks, we have removed the Metadata block and will commit.

**Comment by v-gsapaharam** [comment]
> Check the pipeline results for changes that need to be made... Also, remove the unused pngs

This has been taken care of

**Comment by bmoore-msft** [comment]
@v-gsapaharam - I'm still seeing pipeline errors and all the extraneous PNG files.

**Comment by v-gsapaharam** [comment]
> @v-gsapaharam - I'm still seeing pipeline errors and all the extraneous PNG files.

Need some pointers, how we can make svg files reference in "Readme.md" specific to our template

**Comment by v-gsapaharam** [comment]
> Check the pipeline results for changes that need to be made... Also, remove the unused pngs

@bmoore-msft - All pipeline/build issues resolved, could you please review once.

**Comment by bmoore-msft** [comment]
Closing abandoned request - please reopen if you can get a passing ci

**Pull request closed without merging**